### PR TITLE
Ag-grid odyssey theme characteristics

### PR DESCRIPTION
**Pull request by ZarkoPernar**

### CODE REVIEW

File: packages/ag-grid-theme/css/odyssey-ag-theme.scss [hunk 1]
```diff
@@ -453,18 +490,18 @@ $ag-theme-odyssey-params-touch-dark: map-merge(
...
+    var(--uitk-selectable-foreground)
   );
   --agGrid-checkbox-foreground-selected: var(
     --uitkAgGrid-checkbox-foreground-selected,
-    var(--uitk-text-primary-foreground)
+    var(--uitk-selectable-primary-foreground-selected)
   );
-  --_agGrid-checkbox-border: var(--uitk-selectable-borderWidth)
-    var(--uitk-selectable-borderStyle) var(--uitk-selectable-borderColor);
+  --_agGrid-checkbox-border: 1px var(--uitk-selectable-borderStyle)
```
Comment (feliciackh) [inline packages/ag-grid-theme/css/odyssey-ag-theme.scss:503]:
```suggestion
  --agGrid-checkbox-border: 1px var(--uitk-selectable-borderStyle)
```

File: packages/ag-grid-theme/css/odyssey-ag-theme.scss [hunk 2]
```diff
@@ -453,18 +490,18 @@ $ag-theme-odyssey-params-touch-dark: map-merge(
...
+    var(--uitk-selectable-borderColor);
   --agGrid-checkbox-border: var(
     --uitkAgGrid-checkbox-border,
     var(--_agGrid-checkbox-border)
```
Comment (feliciackh) [inline packages/ag-grid-theme/css/odyssey-ag-theme.scss:507]:
```suggestion
    var(--agGrid-checkbox-border)
```

File: packages/ag-grid-theme/css/odyssey-ag-theme.scss (hunk 1 above)
Comment (ZarkoPernar) [reply packages/ag-grid-theme/css/odyssey-ag-theme.scss:503]: This is a "private" css variable so the underscore was intentional.

**Review by feliciackh — DISMISSED** [review]
lgtm

File: packages/ag-grid-theme/css/_salt-ag-theme-mixin.scss [hunk 3]
```diff
@@ -1,32 +1,40 @@
...
+
   .ag-ltr,
   .ag-rtl {
     .ag-has-focus .ag-cell-focus:not(.ag-cell-range-selected),
     .ag-context-menu-open .ag-cell-focus:not(.ag-cell-range-selected),
     .ag-has-focus .ag-full-width-row.ag-row-focus .ag-cell-wrapper.ag-row-group,
     .ag-cell-range-single-cell,
     .ag-cell-range-single-cell.ag-cell-range-handle {
-      border: 2px dotted var(--agGrid-color-focus);
+      border: 2px dotted var(--agGrid-color-focus) !important;
```
Comment (libertymayc) [inline packages/ag-grid-theme/css/_salt-ag-theme-mixin.scss:27]: --salt-editable-borderWidth-active 2px
--salt-focused-outlineStyle dotted
can use if desired

File: packages/ag-grid-theme/css/salt-ag-theme.scss [hunk 4]
```diff
@@ -1,9 +1,46 @@
...
-.ag-theme-odyssey-medium-light,
-.ag-theme-odyssey-low-light,
-.ag-theme-odyssey-touch-light {
+div[class*="ag-theme-salt-high"] {
+  --agGrid-row-height-default: 25px;
+  --agGrid-padding-default: 8px;
+  --agGrid-fontSize-default: 11px;
+  --agGrid-header-fontSize-default: 10px;
+  --agGrid-separatorGap-default: 8px;
+  --agGrid-cornerTag-size-default: 8px;
```
Comment (libertymayc) [inline packages/ag-grid-theme/css/salt-ag-theme.scss:36]: we usually avoid -default on the end of tokens but you can decide if necessary for this case

Comment (ZarkoPernar) [reply packages/ag-grid-theme/css/salt-ag-theme.scss:36]: I c/p this from grid.css. Not even using most of these, so I might just delete them.

File: packages/ag-grid-theme/css/_salt-ag-theme-mixin.scss (hunk 3 above)
Comment (ZarkoPernar) [reply packages/ag-grid-theme/css/_salt-ag-theme-mixin.scss:27]: Nice, thanks

File: packages/ag-grid-theme/css/_salt-ag-theme-mixin.scss [hunk 5]
```diff
@@ -123,12 +131,94 @@
...
+  }
+
   .ag-row-selected {
-    //z-index: 2;
-    border: transparent;
     background-color: var(--agGrid-row-background-selected);
-    //border-bottom: 0 solid var(--agGrid-row-borderColor-selected);
-    //outline: 1px solid var(--agGrid-row-borderColor-selected);
+    border-color: var(--agGrid-row-borderColor-selected);
+    // position: relative;
```
Comment (alycrys) [inline packages/ag-grid-theme/css/_salt-ag-theme-mixin.scss:161]: don't think you need this commented code

File: .gitignore [hunk 6]
```diff
@@ -51,3 +51,4 @@ yarn-error.log*
 # Autogenerated CSS
 /packages/ag-grid-theme/uitk-ag-theme.css
 /packages/ag-grid-theme/odyssey-ag-theme.css
```
Comment (alycrys) [inline .gitignore:53]: can this line be removed now?

File: packages/ag-grid-theme/stories/examples/CellDropdownEditor.tsx [hunk 7]
```diff
@@ -37,7 +37,7 @@ const CellDropdownEditor = (props: AgGridReactProps) => {
   ]);
 
   const { isGridReady, agGridProps, containerProps, api } = useAgGridHelpers(
-    isNewTheme ? "ag-theme-odyssey" : undefined
+    isNewTheme ? "ag-theme-salt" : undefined
```
Comment (alycrys) [inline packages/ag-grid-theme/stories/examples/CellDropdownEditor.tsx:40]: I'd personally rename 'isNewTheme' to 'isSaltTheme'

File: packages/ag-grid-theme/stories/ag-grid.doc.stories.mdx [hunk 8]
```diff
@@ -69,37 +69,37 @@ For dark mode,
...
 
-### <a name="salt"></a>Installing Salt AG Grid Theme (New theme)
+### <a name="salt"></a>Installing Salt AG Grid theme
 
-To import the Salt AG Grid Theme, run the following command:
+To import the Salt AG Grid theme, run the following command:
 
 ```
-import '@salt-ds/ag-grid-theme/odyssey-ag-theme.css';
+import '@jpmorganchase/uitk-ag-grid-theme/salt-ag-theme.css';
```
Comment (alycrys) [inline packages/ag-grid-theme/stories/ag-grid.doc.stories.mdx:77]: shouldn't this be `import '@salt-ds/ag-grid-theme/salt-ag-theme.css';`?

Comment (feliciackh) [reply packages/ag-grid-theme/stories/ag-grid.doc.stories.mdx:77]: Great spot - think some merges messed up and reverted this name back, will update this in my branch (off of Zarko's)

File: .gitignore (hunk 6 above)
Comment (feliciackh) [reply .gitignore:53]: yes will do!

File: packages/ag-grid-theme/stories/examples/CellDropdownEditor.tsx (hunk 7 above)
Comment (feliciackh) [reply packages/ag-grid-theme/stories/examples/CellDropdownEditor.tsx:40]: good idea, i agree - will update it now. Might as well

**Pull request merged**